pkg/nsx/services/vpc/vpc.go

```diff
@@ -50,9 +51,9 @@ type VPCService struct {
 	LbsStore *LBSStore
 }
 
-func (s *VPCService) GetDefaultNetworkConfig() (*v1alpha1.VPCNetworkConfiguration, error) {
+func GetDefaultNetworkConfig(client client.Client) (*v1alpha1.VPCNetworkConfiguration, error) {
 	vpcNetworkConfigList := &v1alpha1.VPCNetworkConfigurationList{}
-	err := s.Client.List(context.Background(), vpcNetworkConfigList)
+	err := client.List(context.Background(), vpcNetworkConfigList)
```

Comment:
Is this changed for test?

Reply:
I was thinking about calling this function in bindingmap controller without introducing the dependency of vpc. But I feel it might be better to introduce the dependency. Removed those changes, thanks